clients/context-registry/src/ContextRegistryClient.ts

```diff
@@ -158,6 +158,9 @@ export class ITwinAccessClient extends WsgClient implements ITwinAccess {
     const projectQuery = {...queryOptions};
     const assetQuery = {...queryOptions};
 
+    // Known skip inaccuracy when skipping beyond the number elements of one of the two endpoints
+    //  this issue is known and ignored as this feature is only used by ui-test-apps
+    //  as such a small level of inaccuracy is acceptable for now
```

Comment:
ContextRegistryClient is a publicly available package, it is not only used in ui-test-apps:
https://www.itwinjs.org/reference/context-registry-client/contextregistry/contextregistryclient/

Reply:
The ContextRegistryClient is going to go away as it stands today. There will be a replacement client that specifically targets the Projects API directly and will thus correctly implement whatever required.

The interface we're creating within iTwin.js Core is just the pieces needed for the Core itself, not every use-case there could be for the client. That will be the responsibility of the client, and whomever creates it, to decide what functionality is exposed and how.